class CustomerAlreadyExists(Exception):
    pass


class CustomerNotFound(Exception):
    pass
